mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.tmpl

```diff
@@ -549,6 +549,23 @@ func ResourceStorageBucket() *schema.Resource {
 					},
 				},
 			},
+			"hierarchical_namespace": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
```

Comment:
I think the whole block should be configured with ForceNew behavior. This block can not be changes or remove from the config once it is configured, Right?

Reply:
Yes updated block with ForceNew